cmd/kpod/commit.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	commitFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "change, c",
+			Usage: "Apply the following possible instructions to the created image (default []): CMD | ENTRYPOINT | ENV | EXPOSE | LABEL | STOPSIGNAL | USER | VOLUME | WORKDIR",
+		},
+		cli.StringFlag{
+			Name:  "message, m",
+			Usage: "Set commit message for imported image",
+		},
+		cli.StringFlag{
+			Name:  "author, a",
+			Usage: "Set the author for the image comitted",
+		},
+		cli.BoolTFlag{
+			Name:  "pause, p",
+			Usage: "Pause container during commit",
+		},
+	}
+	commitDescription = `Create an image from a container's changes.
+	Optionally tag the image created, set the author with the --author flag,
+	set the commit message with the --message flag,
+	and make chnages to the instructions with the --change flag.`
+	commitCommand = cli.Command{
+		Name:        "commit",
+		Usage:       "Create new image from a container's changes",
+		Description: commitDescription,
+		Flags:       commitFlags,
+		Action:      commitCmd,
+		ArgsUsage:   "CONTAINER [REFERENCE]",
```

Comment:
What is a REFERENCE?

Reply:
REFERENCE is like image-name:tag, changed it to [REPOSITORY[:TAG]] like docker does.